shared/teams/team/header/index.tsx

```diff
@@ -1,31 +1,31 @@
-// @flow
 import * as React from 'react'
 import * as Constants from '../../../constants/teams'
 import * as Types from '../../../constants/types/teams'
 import * as Kb from '../../../common-adapters'
 import AddPeopleHow from './add-people-how/container'
-import type {Response} from 'react-native-image-picker'
+// @ts-ignore not typed yet
+import {Response} from 'react-native-image-picker'
 import NameWithIconWrapper from './name-with-icon-wrapper'
 import * as Styles from '../../../styles'
 
 export type Props = {
-  canChat: boolean,
-  canEditDescription: boolean,
-  canJoinTeam: boolean,
-  canManageMembers: boolean,
-  description: string,
-  loading?: boolean,
-  memberCount: number,
-  openTeam: boolean,
-  role: Types.MaybeTeamRoleType,
-  showingMenu: boolean,
-  teamname: Types.Teamname,
-  onAddSelf: () => void,
-  onChat: () => void,
-  onEditDescription: () => void,
-  onEditIcon: (image?: Response) => void,
-  onFilePickerError: (error: Error) => void,
-  onRename: ?() => void,
+  canChat: boolean
+  canEditDescription: boolean
+  canJoinTeam: boolean
+  canManageMembers: boolean
+  description: string
+  loading?: boolean
+  memberCount: number
+  openTeam: boolean
+  role: Types.MaybeTeamRoleType
+  showingMenu: boolean
+  teamname: Types.Teamname
+  onAddSelf: () => void
+  onChat: () => void
+  onEditDescription: () => void
+  onEditIcon: (image?: Response) => void
+  onFilePickerError: (error: Error) => void
+  onRename: () => void | null
```

Comment:
looks like the bug recently fixed in `flow-to-ts`. should be `(() => void) | null`